projects/05-fleet-management-api/project.yml

```
@@ -1,10 +1,43 @@
 track: web-dev
 learningObjectives:
-  - java
-  - python
-  - object-oriented-programming
-  - node
   - sql
   - db
   - postgres/data-types
   - postgres/indexes
+
+# permite mayor control de la informacion en seleccion de proyecto
+# se pretende tener una semantica predefinida q controle el comportamiento
+# de la interfaz.
+# por ahora estos son los unicos valores permitidos
+tags:
+  - featured
+  - beta
+  - deprecated
+  - hidden
+
+# para proyectos que tienen diferentes implementaciones (aka rutas)
+variants:
+  - name: java
+    learningObjectives:
+      - java
+      - object-oriented-programming
+  - name: node
+    learningObjectives:
+      - node
+      # Cualquier OA puede expresarse como objeto, por ejemplo para flaggear
+      # como "opcional".
+      # por ahora es la unica propiedad soportada.
+      - object-oriented-programming:
+          optional: true
+  - name: python
+    learningObjectives:
+      - python
+      - object-oriented-programming:
+          optional: true
```

me sumo al comentario de que, para agregar la funcionalidad de mostrar o no un OA, no es necesario que sea un objeto que incluya una propiedad `optional`, sino que simplemente puede ser un booleano, es decir:

```suggestion
      - object-oriented-programming: true
```

---

@mfdebian ojo q los casos son 2 distintos:
- uno es cuando el OA no se debe mostrar
- el segundo es q se debe mostrar pero no es obligatorio conseguirlo para dar el proyecto como terminado

Dicho eso, asumiría q:
```yml
- object-oriented-programming: true
```
tendría el mismo efecto q:
```yml
- object-oriented-programming
```